main/ui/ui.go

```diff
@@ -84,6 +84,16 @@ func main() {
 		TimeToLive:   time.Minute * 5, // Cache items invalidated after 5 minutes.
 		RequestLimit: 500,
 	})
+	for i := 0; i < 10; i++ {
```

Comment:
It's arbitrary, I guess. Previously there was an explosion of goroutines doing background fetches, but you probably only need a handful.

You can handle 1 query per goroutine, so at "X" QPS (counting number of metrics named in MQE queries), the latency has to be at most 10 / X seconds to keep up.

So with 100 QPS, each goroutine has to handle 10 requests per second, capping you at 100ms maximum latency to keep up (or some background cache updates will be dropped).

This code isn't actually used in the Square internal repo, so this being non-configurable at the moment is not such a big deal.